tasks/csgitbot.yaml

```
@@ -0,0 +1,138 @@
+- name: Install acl, git, python3-pip and python3-venv
+  become: yes
+  ansible.builtin.package:
+    name:
+      - acl 
+      - git
+      - python3-pip
+      - python3-venv
+    state: latest
+    update_cache: yes
+
+- name: Create the bot user
+  become: yes
+  ansible.builtin.user:
+    state: present
+    system: true
+    name: bot
+    home: /opt/bot
+    shell: /usr/sbin/nologin
+
+- name: Get Python version
+  ansible.builtin.command:
+    cmd: python3 --version
+  register: python_version
+
+- name: Set Python version variable
+  ansible.builtin.set_fact:
+    python_version: "{{ python_version.stdout_lines[0] }}"
+
+- name: Print Python version
+  ansible.builtin.debug:
+    var: python_version
+
+- name: Clone a github repository
+  become: yes
+  become_user: bot
+  ansible.builtin.git:
+    repo: "https://github.com/csunibo/csgitbot"
+    dest: /opt/bot/source
+    version: main
+    force: yes
+  register: git_result
+
+- name: check if virtual environment exists
```

Roba piccolissima: se riesci a mettere la maiuscola su "check"
```suggestion
- name: Check if virtual environment exists
```